docs/mp/guides/overview.adoc

```diff
@@ -97,7 +97,48 @@ in your Helidon MP application.
 --
 Learn how to build a Helidon MicroProfile (MP) application from scratch.
 --
-====
+
+[CARD]
+.Helidon MP Upgrade guide
+[link=migration.adoc]
+--
+Learn how Upgrade your Helidon MP application.
```

Comment:
how _to_ upgrade. Or how to migrate from XXX to XXX.

Reply:
Fixed.